### ISSUE TITLE
Storm-cli-utilities requires storm-counterexamples

### ISSUE THREAD

**Issue by volkm** [moves-rwth/storm#209]
The file `model-handling.h` requires the library `storm-counterexamples` for e.g. printing counterexamples.
I am wondering if we could remove this dependency and move the support for counterexamples into the dedicated library.

**Comment by sjunges** [comment]
I looked at this as I now also added the dedicated gamebased-ar. The storm-cli-utilities are utilities for cli and code for the main storm lib. I think what would make sense is to make seperate folder for storm-cli which contains the model-handling.h and have a storm-cli-utilities that eventually contains stuff for the clis, such as printing and the settings.

**Comment by volkm** [comment]
I agree, storm-cli would then be the directory for the main storm binary and we could also move `src/storm/storm.cpp` into there.
